src/components/Videos.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import { Grid, Embed } from 'semantic-ui-react'
+
+const Videos = (props) => {
+  return (
+    <Grid.Column width={4} className={props.id}>
```

Comment:
@nataliyah123 Do we want the `Videos` component to always occupy 4 columns? It's not a big deal but it would be nice if the parent component rendering `Videos` controlling the layout.